Runtime/Components/Cinemachine3rdPersonFollow.cs

```diff
@@ -121,12 +134,11 @@ public override void MutateCameraState(ref CameraState curState, float deltaTime
 
         void PositionCamera(ref CameraState curState, float deltaTime)
         {
-            var targetPos = FollowTargetPosition;
```

Comment:
FollowTargetPosition is a function call.  It makes sense to grab it here instead of calling it multiple times.  This cleanup is too zealous!

This drives home what Cort was saying in the last DOTS townhall.  Function calls that look like simple variable reads are misleading and dangerous.

Reply:
OMG fell into that trap right away. Thanks!!!